### ISSUE TITLE
build.sbt, missing semicolumn at the end of pomdetails

### ISSUE THREAD

**Issue by Ofthestreet** [norbert-radyk/spoiwo#11]
Hi,

I am a noob with sbt. I installed the last sbt version 0.11.3. When I tried to run the build using sbt, I get an error:
```
cdelarue@host /cygdrive/u/spoiwo-master
$ "d:/Program Files (x86)/sbt/bin/sbt"
[error]
[error] [U:\\spoiwo-master\build.sbt]:22: ';' expected but 'val' found.
[error] Probably problem with parsing xml group, please add parens or semicolons                                                                                          :
[error] Replace:
[error] val xmlGroup = <a/><b/>
[error] with:
[error] val xmlGroup = (<a/><b/>)
[error] or
[error] val xmlGroup = <a/><b/>;
[error]
[error]
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? q
```
After adding the requested trailing ; at the end of pomdetails, sbt becomes must more happy :D

**Comment by norbert-radyk** [comment]
Hi, 
Any reason for using 3 year old version of SBT, rather than going for the latest one? As far as I remember I have used 0.13.8 and it worked fine. Will double check this one in the next couple of days. 

Thanks, Norbert

**Comment by Ofthestreet** [comment]
You are right, I was wrong when indicating the version of sbt 
```
> about
[info] This is sbt 0.13.8
[info] The current project is {file:/U:/spoiwo-master/}spoiwo 1.0.7
[info] The current project is built against Scala 2.10.5
[info] Available Plugins: sbt.plugins.IvyPlugin, sbt.plugins.JvmPlugin, sbt.plugins.CorePlugin, sbt.plugins.JUnitXmlReportPlugin
[info] sbt, sbt plugins, and build definitions are using Scala 2.10.4
> quit
[error] Not a valid command: quit (similar: exit, about)
[error] Not a valid project ID: quit
[error] Expected ':' (if selecting a configuration)
[error] Not a valid key: quit
[error] quit
[error]     ^
>

cdelarue@host /cygdrive/u/spoiwo-master
$ vi build.sbt

cdelarue@host /cygdrive/u/spoiwo-master
$ "d:/Program Files (x86)/sbt/bin/sbt" compile
[error] [U:\spoiwo-master\build.sbt]:22: ';' expected but 'lazy' found.
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore?
```

**Comment by norbert-radyk** [comment]
SBT has been fixed as part of working on release/1.1.0
